go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/r0busta/go-shopify-graphql/v9
```

Comment:
should stay as is if you want it to work
need to fork and use the fork otherwise